mcs/build/profiles/monodroid_designer.make

```diff
@@ -0,0 +1,36 @@
+#! -*- makefile -*-
```

Comment:
This is a carbon copy of `monodroid.make`

Reply:
Changes were made: 

* The API_BIN_PROFILE was changed to 4.7.1 in order to find `System.Configuration`
* The `PLATFORM_MACOS` conditional was added to allow for a proper macos build